backend/src/main/java/ca/bc/gov/hlth/accounttransfer/controller/AccountsController.java

```diff
@@ -187,7 +196,7 @@ private void transferOrganization(String userId, OrgDetails ldapOrg, User user,
 		ObjectMapper mapper = new ObjectMapper();
 		try {
 			// Check if the org is already assigned
-			if (organizationExists(orgDetails, ldapOrg)) {
+			if (!orgDetails.isEmpty()) {
```

Comment:
I don't think it will be possible to have non empty orgDetails at this point because of the validation we're adding. We can remove this conditional and associated logging.